architecture/core_objects/builds.adoc

```diff
@@ -10,30 +10,78 @@
 toc::[]
 
 == Overview
-A build is a process of transforming input parameters, typically source code, into a resulting object, typically a runnable image.
+A build is the end result of transforming input parameters, typically source code, into a resulting object, typically a runnable image.
+
+For a list of build commands, see the
+link:../../dev_guide/builds.html[Developer's Guide].
 
 == BuildConfig
-The `BuildConfig` object is the definition of the entire build process. It consists of the following elements:
+The `BuildConfig` object is the definition of the entire build process, and is dictated by link:../../dev_guide/builds.html#build-triggers[build triggers]:
 
-[horizontal]
-triggers:: Define policies used for automatically invoking builds:
-GitHub webhooks::: GitHub specific webhooks that specify which repository
+====
+----
+apiVersion: v1beta3
```

Comment:
just a note, there will be `v1` soon, but I think this applies to many other examples we have in docs... don't change it now, just a reminder